### ISSUE TITLE
yesterdays update broke the login

### ISSUE THREAD

**Issue by harakiwi** [duplicati/duplicati#2200]
tl;dr: Looks like my install died... Anyways it's 2016 - things keep dying left right and center, Happy Holidays!

long:

I have:
- [x] haven't found anything related to this weeks updates
- [x] haven't found anything related to the black/white icon
- [x] haven't found anything related to the broken login
- [x] haven't found anything related to this 3 things combined

----------------------------------------

### Version info
Duplicati Version: 2.0.1.36 **i guess**
Operating System: **Win 10 x64 Pro**
Backend: **ACD**

### Bug description
Updated after my backups finished via duplicati's "activate" but this time the service came back the login isn't working. Remote-Logged on to my file server and found a black and white duplicati icon instead the green and white outlined one. Repaired the install via MSI Installer and now the icon is back to normal, still no login working. Updated manually, still not working. Restarted the server, too.
Web interface is broken, my saved password isn't working; entering no password doesn't get any error message

### Steps to reproduce
- updated this an hour ago
- found the bug
- repaired the installation
- updated manually
(- disabled windows firewall and any security software (AVAST Free))

**Actual result:** Can't access the web interface at all / broken.
**Expected result:** Well the last 4-5 updates worked fine...

### Screenshots
i can post you the green login screen if you want :/

### debug log
can't access any logs

**Comment by agrajaghh** [comment]
related / duplicate of #2186

**Comment by CountBigBang** [comment]
I just updated from 2.0.34 to 2.0.36 and had the same issue, i.e. I can't login. I am glad to hear that it seems to be "just" a UI issue. But to my knowledge there is no other way to access the files or the configuration or the logs, is there? So when can we expect the 2.1 release? Right now, Duplicati is unusable.

**Comment by kenkendk** [comment]
I have added an updated version `2.0.1.38` now, that you can install to fix this issue.
You can either install via the installers:
https://github.com/duplicati/duplicati/releases/tag/v2.0.1.38-2.0.1.38_canary_2016-12-27

Or use the built-in updater.
If you need to access the user interface, you can either delete the `2.0.1.36` build from the Updates folder, and start with `2.0.1.35` or you can use the commandline option `--webservice-password=` to set a blank password.

**Comment by harakiwi** [comment]
thanks!

 --webservice-password= was what i was searching for 👍 

Looks like nothing broke. Now on to start verifying my tasks ;)
